styleguide.config.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
 	components: 'src/components/*/*.vue',
 	getComponentPathLine(componentPath) {
 		const name = path.basename(componentPath, '.vue')
-		return `import { ${name} } from '@nextcloud/vue/dist/Components/${name}'`
+		return `import ${name} from '@nextcloud/vue/dist/Components/${name}'`
```

Comment:
is this related to the UserBubble component?

Reply:
Yes and no!
This is related to all components, I noticed the import text was wrong, we cannot import with the brackets if we use the direct component path.
I will move this to another pr for better clarity, let's not use this as a free pass!